ch07.adoc

```diff
@@ -723,14 +725,554 @@ data:  // time coordinates translated to datetime format
+
+In the convention of this section, a data variable is described as an anomaly by including in its **`cell_methods`** attribute a string of the form "__name__**`:`** [__name__**`:`** ...] **`anomaly wrt`** __norm__".
+This **`cell_methods`** entry tells us:
+
+* that the data variable contains anomalies (__A__ in the notation of the start of this section) calculated from data (__P__) with the same dimensions, as the difference from the normal statistic (__N__) named __norm__.
+
+* at which stage the anomaly __A__ = __P__ &minus; __N__ was calculated.
+This is known because the entries in **`cell_methods`** appear in order of application.
+
+* that __N__ was calculated from the variation of __P__ over dimensions identified by the __name__(s).
+
+Each __name__ must identify an "axis" of the anomaly data variable __A__.
+By "axis" we mean a dimension and its corresponding coordinate variable, or a scalar coordinate variable.
+In either case, we refer to the axis as an "anomaly axis" and to the variable as an "anomaly coordinate variable".
+The anomaly coordinate variable of each axis must have a **`standard_name`** attribute.
+Usually there is only one anomaly axis, and usually it is a spatiotemporal axis.
+
+For instance, for a data variable containing anomalies with respect to the zonal mean, __name__ identifies longitude as the anomaly axis e.g. "**`longitude: anomaly wrt`** __norm__".
+For an anomaly with respect to the area minimum, we need two __name__s e.g. "**`lat: lon: anomaly wrt`** __norm__".
+As described in <<cell-methods>>, the combination of horizontal axes can alternatively be represented by the word **`area`**, thus "**`area: anomaly wrt`** __norm__".
+
+For an anomaly with respect to a statistic computed over time or climatological time, __name__ identifies the time axis of the anomaly data variable.
+
+There are two alternatives for __norm__.
+In both cases, __norm__ is an ancillary variable of the anomaly data variable (<<ancillary-data>>).
+
+If the norm data variable __N__ is present in the dataset, it can be named as __norm__ in the **`cell_methods`** of the anomaly variable.
+Such would be the case in a dataset that contains both zonal means and anomalies relative to those means.
+No modification to the metadata of __N__ is required for it to serve as the norm for __A__, and __N__ may still be treated as a data variable in its own right as well.
+We describe the use of a norm data variable first (<<anomalies-norm-data>>) because this case is conceptually more obvious, although it is uncommon for __N__ to be present in the dataset.
+
+The second alternative (<<anomalies-norm-metadata>>) is where __norm__ in **`cell_methods`** identifies a "norm metadata variable" instead of the norm data variable.
+A norm metadata variable contains information about the norm axes, but no data of its own.
+This method can be used regardless of whether the norm data variable is present in the dataset as well.
```

Comment:
"present in the dataset as well" sounds weird to me.  I would prefer "also present in the dataset."

Reply:
OK, changed.